modules/apps/object/object-test/src/testFunctional/tests/ObjectRootModel.testcase

```diff
@@ -0,0 +1,151 @@
+				fieldName = "textField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "Parent");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "Parent",
+				objectName_2 = "Child",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+		}
+
+		task ("and Given Child is bound to Parent") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectRootModel.bindToRootObject(
+				childObject = "Child",
+				rootObjectPath = "Parent (Relationship)");
+		}
+
+		task ("When the user navigates to Child") {
+			ObjectPortlet.selectCustomObject(label = "Child");
+		}
+
+		task ("Then the enable widget option and panel key category are read only") {
+			AssertElementPresent(
+				key_checkboxName = "Show Widget in Page Builder",
+				locator1 = "Checkbox#CHECKBOX_DISABLED");
+
+			AssertElementPresent(
+				key_text = "Control Panel > Object",
+				locator1 = "Dropdown#ANY_DISABLED");
+		}
+
+		task ("and When the user publishes Parent") {
+			ObjectAdmin.publishObjectViaAPI(objectName = "Parent");
+
+			Navigator.gotoBack();
+		}
+
+		task ("Then Child is published as well and does not have a dedicated portlet") {
+			AssertTextEquals(
+				key_label = "Child",
+				locator1 = "ObjectAdmin#OBJECT_STATUS",
+				value1 = "Approved");
+
+			Click(locator1 = "ApplicationsMenu#TOGGLE");
+
+			AssertElementNotPresent(
+				key_category = "Objects",
+				key_portlet = "Child",
+				locator1 = "ApplicationsMenu#PORTLET");
+		}
+	}
+
+	@description = "LPS-189380 - Verify the user can to see relationships tabs in the initial out-of-the-box layout of one level of children"
+	@priority = 4
+	test CanViewRelationTabOOTB {
+		task ("Given two custom object definitions, Santa Cruz and Andre, with a text field and a 1:M relationship between them") {
+			for (var objectName : list "Andre,SantaCruz") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = ${objectName},
+					objectName = ${objectName},
+					pluralLabelName = ${objectName});
+			}
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Type your name to become a Santa Cruz fan",
+				fieldName = "textField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "SantaCruz");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "SantaCruz",
+				objectName_2 = "Andre",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+		}
+
+		task ("and Given Andre is bound to Santa Cruz") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectRootModel.bindToRootObject(
+				childObject = "Andre",
+				rootObjectPath = "SantaCruz (Relationship)");
+		}
+
+		task ("and Given an object entry in Santa Cruz") {
+			ObjectAdmin.publishObjectViaAPI(objectName = "SantaCruz");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "textField",
+				objectName = "SantaCruze",
```

Comment:
It's not a typo. The plural label for this object in the API is SantaCruzes. As this macro adds an "s" to the value we give to objectName (See [here](https://github.com/davisaints/liferay-portal/blob/4f33caca1b181bc1a1a7968773ce7447f79449e7/modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro#L595)), I only needed to put SantaCruze.